be/src/main/java/codesquad/issuetracker/service/IssueService.java

```diff
@@ -20,50 +35,152 @@
+
+        Reply reply = Reply.createReply(issue, writer, form.getComment());
+        issue.addReply(reply);
+
+        List<Long> assigneeIds = form.getAssigneeIds();
+        for (Long assigneeId : assigneeIds) {
+            issue.addAssignee(Assignee.createAssignee(issue, getMemberById(assigneeId)));
+        }
+
+        List<Long> labelIds = form.getLabelIds();
+        for (Long labelId : labelIds) {
+            issue.addIssueLabel(IssueLabel.createIssueLabel(issue, getLabelById(labelId)));
         }
 
-        return new long[]{countOfOpenIssues, countOfClosedIssues};
+        issueRepository.save(issue);
+
+        return IssueDto.from(issue);
     }
 
     @Transactional
-    public void updateStatusByIssueId(IssueStatusUpdateForm updateForm) {
-        issueRepository.update(updateForm.getUpdatedStatus(), updateForm.getIdOfIssues());
+    public void updateStatusOfIssue(IssueStatusUpdateForm form) {
+        List<Issue> issues = issueRepository.findAllById(form.getIdOfIssues());
+        IssueStatus updatedStatus = form.getUpdatedStatus();
+        issues.forEach(issue -> issue.updateStatus(updatedStatus));
+    }
+
+    @Transactional
+    public void updateSubjectOfIssue(Long id, String subject) {
+        Issue issue = getIssueById(id);
+
+        issue.updateSubject(subject);
+    }
+
+    @Transactional
+    public void updateMilestoneOfIssue(Long id, Long milestoneId) {
+        Issue issue = getIssueById(id);
+
+        if (milestoneId == null) {
+            issue.updateMilestone(null);
+            return;
+        }
+
+        Milestone milestone = getMilestoneById(milestoneId);
+
+        issue.updateMilestone(milestone);
+    }
+
+    @Transactional
+    public void updateLabelsOfIssue(Long id, List<Long> ids) {
+        Issue issue = getIssueById(id);
+        issueLabelRepository.deleteByIssueId(issue.getId());
+
+        if (ids.size() == 0) {
+            return;
+        }
+
+        labelRepository.findAllById(ids)
+            .forEach(label -> issue.addIssueLabel(IssueLabel.createIssueLabel(issue, label)));
+    }
+
+    @Transactional
+    public void updateAssigneeOfIssue(Long id, List<Long> ids) {
+        Issue issue = getIssueById(id);
+        assigneeRepository.deleteByIssueId(issue.getId());
+
+        if (ids.size() == 0) {
+            return;
+        }
+
+        memberRepository.findAllById(ids)
+            .forEach(member -> issue.addAssignee(Assignee.createAssignee(issue, member)));
+    }
+
+    @Transactional
+    public void delete(Long id) {
+        issueRepository.deleteById(id);
+    }
+
+    private Issue getIssueById(Long id) {
+        return issueRepository.findById(id).orElseThrow(() -> {
+            throw new IllegalStateException("존재하지 않는 이슈입니다.");
+        });
+    }
+
+    private Member getMemberById(Long id) {
+        return memberRepository.findById(id).orElseThrow(() -> {
+            throw new IllegalStateException("존재하지 않는 회원입니다.");
+        });
+    }
```

Comment:
메소드 이름에 null을 반환하는지, throw 하는지 선언해주면 메소드를 사용하는 쪽에서 동작을 예측하기 더 쉬워집니다.
```suggestion
    private Issue getIssueByIdOrThrow(Long id) {
        return issueRepository.findById(id).orElseThrow(() -> {
            throw new IllegalStateException("존재하지 않는 이슈입니다.");
        });
    }

    private Member getMemberByIdOrThrow(Long id) {
        return memberRepository.findById(id).orElseThrow(() -> {
            throw new IllegalStateException("존재하지 않는 회원입니다.");
        });
    }
```